doc/classes/VisibleOnScreenEnabler3D.xml

```diff
@@ -1,28 +1,32 @@
 <?xml version="1.0" encoding="UTF-8" ?>
 <class name="VisibleOnScreenEnabler3D" inherits="VisibleOnScreenNotifier3D" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="../class.xsd">
 	<brief_description>
-		Enables certain nodes only when approximately visible.
+		A box-shaped region of 3D space that, when not visible on screen, disables a target node.
 	</brief_description>
 	<description>
-		The VisibleOnScreenEnabler3D will disable [RigidBody3D] and [AnimationPlayer] nodes when they are not visible. It will only affect other nodes within the same scene as the VisibleOnScreenEnabler3D itself.
-		If you just want to receive notifications, use [VisibleOnScreenNotifier3D] instead.
-		[b]Note:[/b] VisibleOnScreenEnabler3D uses an approximate heuristic for performance reasons. It doesn't take walls and other occlusion into account. The heuristic is an implementation detail and may change in future versions. If you need precise visibility checking, use another method such as adding an [Area3D] node as a child of a [Camera3D] node and/or [method Vector3.dot].
-		[b]Note:[/b] VisibleOnScreenEnabler3D will not affect nodes added after scene initialization.
+		[VisibleOnScreenEnabler3D] contains a box-shaped region of 3D space and a target node. The target node will be automatically enabled (via its [member Node.process_mode] property) when any part of this region becomes visible on the screen, and automatically disabled otherwise. This can for example be used to activate enemies only when the player approaches them.
+		See [VisibleOnScreenNotifier3D] if you only want to be notified when the region is visible on screen.
+		[b]Note:[/b] [VisibleOnScreenEnabler3D] uses an approximate heuristic that doesn't take walls and other occlusion into account. This is an implementation detail and may change in future versions.
```

Comment:
This Note is somewhat wrong, it takes Occlusion into account if you are using Occlusion Culling and have set it up properly, it's using the same logic as the Notifier.